engines/director/score.cpp

```diff
@@ -462,15 +471,16 @@ void Score::update() {
 		}
 	}
 
-	byte tempo = _frames[_currentFrame]->_tempo;
-	if (tempo) {
+	byte tempo = _frames[_currentFrame]->_scoreCachedTempo;
+	// puppetTempo is overridden by changes in score tempo
+	if (_frames[_currentFrame]->_tempo || tempo != _lastTempo) {
 		_puppetTempo = 0;
 	} else if (_puppetTempo) {
 		tempo = _puppetTempo;
 	}
 
 	if (tempo) {
-		if (tempo > 161) {
+		if (tempo >= 196) {
```

Comment:
What does happen in D2 and D3?

Reply:
D2 and D3 allow delays of up to 120 seconds, D4 reduces this to 60,  have fixed with a conditional.